pkg/vault/vault_client.go

```diff
@@ -16,7 +16,7 @@ const (
 	yamlDataKey = "yaml"
 )
 
-var vaultURIRegex = regexp.MustCompile(`:[\s"]*vault:[-_\w\/:]*`)
+var vaultURIRegex = regexp.MustCompile(`:[\s"]*vault:[-_.\w\/:]*`)
```

Comment:
@ccojocar, are you aware of any reason not to allow '.' in the regexp?

Reply:
You need to check if vault allows this in its secret path. I think it does according with this https://groups.google.com/forum/#!topic/vault-tool/_E2N0SLryv8

Reply:
thanks @ccojocar. That seems ok then to me.